src/Symfony/Component/Config/Definition/Builder/ArrayNodeDefinition.php

```diff
@@ -239,11 +239,13 @@ public function canBeUnset(bool $allow = true): static
      * enableableArrayNode: {enabled: false, ...}  # The config is disabled
      * enableableArrayNode: false                  # The config is disabled
      *
+     * @param string|null $info A description of what happens when the node is enabled or disabled
+     *
      * @return $this
      */
-    public function canBeEnabled(): static
+    public function canBeEnabled(/* ?string $info = null */): static
     {
-        $this
+        $node = $this
```

Comment:
I would name it `enabledNode` to make the code more readable for next contributors about which node is stored in that variable.

Reply:
I agree with you, I'll take care of it. Thanks for pointing it out!